### DISCUSSION TITLE
Send notification error

### DISCUSSION THREAD

**Discussion by Gonchito** [upptime/upptime#682] [Q&A]
Hello, good morning, I have configured 10 of my own urls and the sending of notifications by email (sendgrid) and telegram.

I turn off one of the urls, run the "Uptime CI" process, it creates the page down incident but doesn't send me any notification prompt. But when I go back to upload the page there if it sends me both notifications advising that it is online again.

I would need it to send me the notifications when it is offline.

Could it be a configuration issue? is it some bug?

Please help, thank you

**Comment by AnandChowdhary** [comment]
That's indeed very weird. Could you share your repository or logs?

**Comment by Gonchito** [comment]
Thanks for answering, where can I copy the log from? I share what I have:

repository:
https://github.com/Gonchito/VerifStatusPages

workflow that created the problem but did not send any notification:
https://github.com/Gonchito/VerifStatusPages/actions/runs/3567225507/workflow

page:
https://gonchito.github.io/VerifStatusPages/

issue created automatically:
https://github.com/Gonchito/VerifStatusPages/issues/21

**Reply by AnandChowdhary** [reply]
You're right, there was an error here: https://github.com/Gonchito/VerifStatusPages/actions/runs/3567225507/jobs/5994654345#step:3:70. Let me try and figure out what was wrong!

**Reply by Gonchito** [reply]
Thank you very much for the answer, could you find the error?

**Reply by AnandChowdhary** [reply]
It got a 404 error when trying to assign someone to this issue: https://api.github.com/repos/Gonchito/VerifStatusPages/issues/21/assignees. Could it be that you deleted the issue manually when it was opened?

**Reply by Gonchito** [reply]
Hello, I was trying to authorize the issue to me from the ".upptimerc.yml" but it looks like it was wrong, I deleted the command but it still doesn't send the notification

**Comment by Gonchito** [comment]
something new?

**Comment by Gonchito** [comment]
it was already solved
